source/styles/main.styl

```diff
@@ -116,14 +116,14 @@ blockquote
     line-height 1.2rem
 
 nav
-  font-size 0.9rem
+  font-size 0.8rem
   margin 70px 0 30px 0
   transition all 0.2s
 
   a
     display inline-block
     line-height 2.2
-    padding 0 15px
+    padding 0 10px
```

Comment:
not really so happy with having the releases link down in that nav.. not really sure, putting links in the right spots is hard.

Maybe in the version box as "view all releases"?